### PR TITLE
FIX: retry to get open orders only for 5 times and do not sync orders…

### PR DESCRIPTION
**Pull request by kbearXD**
… updated in 3 min

### CODE REVIEW

File: pkg/exchange/retry/order.go [hunk 1]
```diff
@@ -47,6 +47,15 @@ func GeneralBackoff(ctx context.Context, op backoff2.Operation) (err error) {
 	return err
 }
 
+func GeneralBackoffLite(ctx context.Context, op backoff2.Operation) (err error) {
+	err = backoff2.Retry(op, backoff2.WithContext(
```
Comment (gx578007) [inline pkg/exchange/retry/order.go:51]: Is it possible to rename `backoff2` to just `backoff`?

**Review by gx578007 — APPROVED** [review]

File: pkg/exchange/retry/order.go [hunk 2]
```diff
@@ -47,6 +47,15 @@ func GeneralBackoff(ctx context.Context, op backoff2.Operation) (err error) {
 	return err
 }
 
+func GeneralBackoffLite(ctx context.Context, op backoff2.Operation) (err error) {
```
Comment (c9s) [inline pkg/exchange/retry/order.go:50]: LiteBackoff

File: pkg/strategy/grid2/active_order_recover.go [hunk 3]
```diff
@@ -89,9 +89,10 @@ func (s *Strategy) recoverActiveOrdersPeriodically(ctx context.Context) {
 func syncActiveOrders(ctx context.Context, opts SyncActiveOrdersOpts) error {
 	opts.logger.Infof("[ActiveOrderRecover] syncActiveOrders")
 
-	notAddNonExistingOpenOrdersAfter := time.Now().Add(-5 * time.Minute)
+	// do not sync orders which is updated in 3 min, because we may receive from websocket and handle it twice
+	doNotSyncAfter := time.Now().Add(-3 * time.Minute)
```
Comment (c9s) [inline pkg/strategy/grid2/active_order_recover.go:93]: is there a better name for this? LOL

Comment (c9s) [reply pkg/strategy/grid2/active_order_recover.go:93]: maybe `syncBefore`?

File: pkg/exchange/retry/order.go (hunk 1 above)
Comment (kbearXD) [reply pkg/exchange/retry/order.go]: simplify it.

**Review by c9s — APPROVED** [review]

**Pull request merged**